### PR TITLE
Media keys with Firefox possible

### PR DESCRIPTION
**Pull request by miicha**
There is an Firefox addon which makes it possible to use media keys to control multimedia playback. With the linked pull request it is possible to control jamstash even running on your own server (just add your server url in the addon options)

### CODE REVIEW

File: dist/settings/settings.html [hunk 1]
```diff
@@ -3,4 +3,4 @@
...
...="https://github.com/tsquillario/Jamstash" target="_blank">https://github.com/tsquillario/Jamstash</a></li> <li>Jamstash Chrome App - <a href="https://chrome.google.com/webstore/detail/jccdpflnecheidefpofmlblgebobbloc" target="_blank">Chrome Web Store</a></li> <li>Beta Site - <a href="http://beta.jamstash.com" target="_blank">http://beta.jamstash.com</a></li> <li><a href="https://twitter.com/JamstashApp" target="_blank">Follow @JamstashApp</a> </li></ul> <h3 class="title">Thanks</h3> <ul class="preferences"> <li>Icons - <a href="http://somerandomdude.com/work/iconic" target="_blank">http://somerandomdude.com/work/iconic</a></li> <li>Audio Library - <a href="http://jplayer.org" target="_blank">http://jplayer.org</a></li> </ul> </div> <div class="clear"></div> <div class="subsection floatleft"> <h3 class="title">Change Log</h3> <ul id="ChangeLog" class="preferences"> <li class="log" ng-repeat="o in changeLog"><span class="version">{{o.date + ' - ' + o.version}}</span> <span class="changes" ng-repeat="a in o.changes" ng-bind-html="a.text"></span> </li> </ul> <a href="" ng-click="changeLogShowMore()">Show More</a> </div> <div class="clear"></div> </div> <div class="clear"></div> </div>
```
Comment (Hyzual) [inline dist/settings/settings.html]: Please change `/app/settings/settings.html`, run `grunt` and then commit.

Files in the `dist/` folder are not meant to be edited, as you can see this is a "minified" HTML file to reduce file transfer time.

**Comment by Hyzual** [comment]
Hi, thanks for contributing to Jamstash !

I've added a comment to the file you modified. Please take it into account and amend the commit and then I'll merge the PR.

By the way, I noticed your link points to a merged PR, perhaps it should point to a download link for the Firefox extension ?

**Comment by miicha** [comment]
I should have read the wiki article before creating the pull request. But now that the environment works as it should I will update it to be "mergeable".
Nevertheless one question remains: The PR is merged but I have no clue when the new addon with this funcionality will be reviewed and released from mozilla. Therefore I would like to link to the firefox addon page and the github repository with the new changes. I hope thats ok with you?

**Comment by Hyzual** [comment]
Hi, as for the previous PR, sorry I took that long. Give me a day or two and this'll be merged too in v4.6.3 :)

**Comment by Hyzual** [comment]
I took the liberty to change the link to the en-US page for the Media Key Firefox addon as I think most of our users know english better than german.
I ended up merging it manually so I'll close the PR.

**Pull request closed without merging**